okrs_2019.md

```diff
@@ -0,0 +1,229 @@
+embed SourceCred results into every open-source collective’s page (e.g. here is
+the [page for Webpack]). This way, collectives can thank their leading
+contributors as well as their financial sponsors. We have enthusiastic buy-in
+from the Open Collective team, and think this could be the start of a great
+partnership, in addition to bringing more attention and users to SourceCred.
+
+To do this, we need to make a number of engineering improvements to SourceCred.
+The mirror module (which loads data from GitHub) needs to be made more robust
+to errors in GitHub's GraphQL implementation ([#996]). We also need a proper
+API for generating cred results, and support for running SourceCred on full
+projects, not just individual repositories. All of this work is vital for any
+SourceCred integrations, rather than being specific to the Open Collective
+integration in particular.
+
+[Open Collective]: https://opencollective.com/
+[page for Webpack]: https://opencollective.com/webpack#contributors
+
+- [ ] Integrate with Open Collective ([#935])
+  - [ ] API-ifyg SourceCred ([#945], [#907])
+  - [ ] Implement 'Project Mode' ([#701])
+  - [ ] Make the SourceCred mirror module robust
+
+[#935]: https://github.com/sourcecred/sourcecred/issues/935
+[#907]: https://github.com/sourcecred/sourcecred/issues/907
+[#945]: https://github.com/sourcecred/sourcecred/issues/945
+[#701]: https://github.com/sourcecred/sourcecred/issues/701
+
+
+### Metrics
+
+We aren't sure what the right metrics are to track engagement and usage by
+downstream projects. For now, we'll measure "engagment", which is defined as:
+the project has taken a public action to use SourceCred, e.g. by linking to the
+cred attribution in their README, adding cred configuration files to their repository,
+or integrating a cred bot on GitHub. Actions that can happen without direct engagement
+from the project (e.g. listing their cred on Open Collective) don't count.
+
+For the purposes of the metrics below, "medium-sized projects" have at least 30 users
+in their cred attribution, and "large-sized projects" have at least 100.
+
+We'd like to see crypto projects start to use SourceCred as a way to transparently
+distribute tokens to contributors. For purposes of the metric, we're tracking public
+announcements or active distributions by well-known projects (i.e. no unknown ICOs).
+
+Finally, we're interested in seeing third-party integrations, of which the Open Collective
+integration is one.
+
+| Metric                                  | Goal | Q4 | Q1 | Q2 | Q3 | Q4 |
+| --------------------------------------- | ---- | -- | -- | -- | -- | -- |
+| # of engaged medium-sized projects      | 50   | 0  |    |    |    |    |
+| # of engaged larged-sized projects      | 5    | 0  |    |    |    |    |
+| # of announced token distributions      | 3    | 0  |    |    |    |    |
+| # of 3rd party integrations             | 3    | 0  |    |    |    |    |
+
+## SourceCred Helps Projects
```

Comment:
The set of KRs below is not actionable enough yet imo. May want to dwindle down to the very specifics (at the risk of picking the wrong ones for now)!

Reply:
This Objective is pretty far in the future; to engage with it, we need to solve a lot of issues like getting users, making cred engaging, adding heuristics so we can really start to try to change behaviors. By the time we're ready to start actually making progress here, we'll have a better idea of what to measure. I'm OK leaving this vague for now. #livingdocument

Reply:
Fair enough. May be worth adding a mention of this is a future objective or forward looking one...